README.md

```diff
@@ -13,17 +13,18 @@ adhocracy+ is designed to make online participation easy and accessible to every
 
 ### Requirements
 
- * nodejs (+ npm)
+ * nodejs (+ npm) (Use Homebrew to reinstall the node in the projects folder runnign: brew install node)
```

Comment:
here good to note that for MacOS it needs homebrew for installing node.
`make install` will take care to install the npm packages for the project, so no need to write down the reinstallation.

Reply:
I took it out! thanks!